hwdb.d/60-keyboard.hwdb

```diff
@@ -28,12 +28,6 @@
 #    <vendor> and <product> are the firmware-provided strings
 #    exported by the kernel DMI modalias, see /sys/class/dmi/id/modalias
 #
-#  - Input driver device name and DMI data match:
```

Comment:
Why are you moving this documentation block around?

I see no reason for this change, please drop it.